src/pycram/robot_description.py

```diff
@@ -247,21 +250,21 @@ def get_manipulator_chains(self) -> List[KinematicChainDescription]:
                 result.append(chain)
         return result
 
-    def get_camera_link(self) -> str:
+    def get_camera_frame(self) -> str:
         """
         Quick method to get the name of a link of a camera. Uses the first camera in the list of cameras.
 
         :return: A name of the link of a camera
         """
-        return self.cameras[list(self.cameras.keys())[0]].link_name
+        return f"{self.name}/{self.get_camera_link()}"
```

Comment:
This return only works if the name for the spawned robot and the name in the robot description are identical

Reply:
changed it to take robot name as an argument.